stata_kernel/stata_magics.py

```diff
@@ -58,10 +57,51 @@ def __init__(self, kernel):
             '-n', dest='n', type=int, metavar='N', default=None,
             help="Execute statement N times per loop.", required=False)
 
+        #######################################################################
+        #                                                                     #
+        #                             %set magic                              #
+        #                                                                     #
+        #######################################################################
+
+        self.set = StataParser(prog='%set', kernel=kernel)
+        # self.set.add_argument(
+        #     '--permanently', dest='perm', action='store_true',
+        #     help="Store settings permanently", required=False)
```

Comment:
Why is this commented out?

Reply:
Ah, this was meant to have a TODO attached; the idea is that if the `--permanently` is passed, this gets set in the `.conf` file, but I wasn't sure what was the best place to put it on (specially since I didn't put a ton of stuff under `%set`